spec/tic_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+require '../lib/tic'
+require '../lib/players'
+def preprocess(indexes)
+  i = 0
+  while i < 2
+    if indexes[i] < 1 or indexes[i] > 3
+      puts 'invalid move only in range of 1-3'
+      return [1, 1]
+    end
+    indexes[i] = indexes[i] - 1 if indexes[i] == 1
+    indexes[i] = indexes[i] + 1 if indexes[i] == 3
+    i += 1
+  end
+  indexes
+end
+
+def gameobj(player)
+  tic = Tic.new
+  tic.players_simble(player)
+  tic
+end
+p1 = Player.new
+p2 = Player.new
+describe Tic do
+  it 'draw combination' do
+    players = [p1, p2]
+    tic = gameobj players
+    draw_pattern = [[1, 1], [2, 1], [2, 2], [3, 3], [3, 1], [1, 3], [2, 3], [3, 2], [1, 2]]
+    i = 1
+    draw_pattern.each do |a|
+      i += 1
+
+      if i == 12 # last iteration where game draws
+        tic.turn(players[i % 2], preprocess(a)).should.equal? 1
+        tic = nil
+        break
+      end
+      tic.turn(players[i % 2], preprocess(a))
+    end
+  end
```

Comment:
This is not an actual way to write the test cases. Check out this example and Refactor your test cases.
![image](https://user-images.githubusercontent.com/38625956/77250521-b495a900-6c72-11ea-8893-2d8177b75399.png)
Check out this [link](https://www.tutorialspoint.com/rspec/rspec_writing_specs.htm) and [link](https://relishapp.com/rspec/docs/gettingstarted).